tests/test_cart.py

```diff
@@ -654,3 +654,20 @@ def test_cart_taxes(request_cart_with_item, shipping_method, vatlayer):
     taxed_price = TaxedMoney(net=Money('8.13', 'USD'), gross=Money(10, 'USD'))
     assert cart.get_shipping_price(taxes=vatlayer) == taxed_price
     assert cart.get_subtotal(taxes=vatlayer) == taxed_price
+
+
+def test_clear_cart_must_be_ajax(rf, customer_user):
+    request = rf.post(reverse('home'))
+    request.user = customer_user
+    request.discounts = None
+    response = clear_cart(request)
+    assert response.status_code == 302
+
+
+def test_clear_cart_check_if_cart_is_empty(request_cart_with_item, rf, client):
```

Comment:
I think we should just call it `test_clear_cart` as it tests the expected behavior.

Also rf seems unused in this function